public/style/components/sponsorship-edit/_index.scss

```diff
@@ -56,3 +56,48 @@
 
   margin-bottom: $standard-padding;
 }
+
+.sponsorship-edit__validation {
+  color: $c-red;
+  font-weight: 300;
+}
+
+.sponsor-tooltip:hover {
+  cursor: help;
+}
+
+.sponsorship-tag {
+  @extend %fs-data-4;
+  display: flex;
+  margin: 0 0 2px 0;
+  border: 1px solid $c-grey-300;
+  padding: 4px 2px 4px 6px;
+  background-color: $c-grey-100;
+  width: fit-content;
+  i {
+    transform: scale(75%);
+    outline: none;
+    border:none;
+    margin-left: 4px;
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
+
+.sponsorship-clash {
```

Comment:
`nit` Some of these classnames are still not meeting convention of having the directory name  i.e. `sponsorship-edit` then the other BEM parts. 
Therefore we would expect `sponsorship-clash` to either be `sponsorship-edit__clash` or  be located in its own folder/file. I would expect the same of `sponsorship-tag` and `sponsor-tooltip`.